docs/general/attributes.md

```diff
@@ -45,6 +49,13 @@ This also covers UDP network interactions where one side initiates the interacti
 In an ideal situation, not accounting for proxies, multiple IP addresses or host names,
 the `server.*` attributes are the same on the client and server.
 
+### Address and port attributes
+
+For the purpose of these semantic conventions, for all IP-based protocols the "address" should be just IP-level address,
+i.e. protocol-specific parts of an address are split into other attributes if applicable such as "port" attributes for
+TCP and UDP. If such transport-specific information is collected and the attribute name does not already uniquely
+identify the transport, then setting the [`network.transport`](#other-network-attributes) is especially encouraged.
```

Comment:
```suggestion
identify the transport, then setting the [`network.transport`](#other-network-attributes) attribute is especially encouraged.
```

Do we want to use any normative wording here? Like :

> identify the transport, then [`network.transport`](#other-network-attributes) SHOULD be set.

Reply:
Let's instead use this:

```suggestion
identify the transport, then setting [`network.transport`](#other-network-attributes) is especially encouraged.
```

I think for normative wording we would really need to add it (or at least a link) to each single attribute. I don't think there is really a need to make it normative either.

Reply:
Sounds good :)